include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp

```diff
@@ -52,6 +52,8 @@ struct alignment_matrix_format
     char const * row_sep{};
     //!\brief row separator symbol (a single symbol)
     char const * row_col_sep{};
+    //!\brief infinity symbol (a single symbol)
+    char const * inf{};
```

Comment:
why don't you make it a char if it is a single symbol?

Reply:
because it is not a single symbol.